app/Livewire/Source/Github/Change.php

```
@@ -28,6 +28,8 @@ class Change extends Component
 
     public ?bool $preview_deployment_permissions = true;
 
+    public ?bool $deployment_statuses_permissions = true;
```

PHPMD (2.15.0)

Avoid excessively long variable names like $deployment_statuses_permissions. Keep variable name length under 20. (undefined)

(LongVariable)